lib/gruf/configuration.rb

```diff
@@ -85,27 +82,24 @@ def reset
       VALID_CONFIG_KEYS.each do |k, v|
         send((k.to_s + '='), v)
       end
+      self.root_path = Rails.root.to_s.chomp('/') if defined?(Rails)
```

Comment:
🍺 It would probably better to have a `gruf-rails` gem to handle framework specific things like this in the future.